src/vl53_main.c

```diff
@@ -0,0 +1,270 @@
+/************************************
+ *       VL53 Main source file      *
+ ************************************/
+
+// Includes
+#include "vl53_main.h"
+#include "hardware_functions.h"
+#include "error_handling.h"
+
+//Global variables
+extern uint16_t PeopleCount;

```

Comment:
In plaats van globaal, zou ik deze misschien liever met ee getter en setter doen. (indien mogelijk)